### PR TITLE
Fix/core integration

### PR DESCRIPTION
**Pull request by theus77**
|Q              |A  |
|---------------|---|
|Bug fix?       |Y|
|New feature?   |N|	
|BC breaks?     |N|	
|Deprecations?  |N|	
|Fixed tickets  |N|

### CODE REVIEW

**Review by laurensmertens — APPROVED** [review]

File: Helper/Environment/Environment.php [hunk 1]
```diff
@@ -80,4 +107,25 @@ public function modifyRequest(Request $request)
...
+        if (!$propertyAccessor->isReadable($this->options, $propertyPath)) {
+            return $default;
+        }
+
+        return $propertyAccessor->getValue($this->options, $propertyPath);
+    }
+
+    public function hasOption(string $option): bool
+    {
+        return isset($this->options[$option]) && null != $this->options[$option];
```
Comment (Davidmattei) [inline Helper/Environment/Environment.php:129]:
```suggestion
        return isset($this->options[$option]) && null !== $this->options[$option];
```

File: Helper/Environment/Environment.php [hunk 2]
```diff
@@ -59,7 +82,11 @@ public function matchRequest(Request $request)
             return true;
         }
 
-        $url = \vsprintf('%s://%s%s', [$request->getScheme(), $request->getHttpHost(), $request->getBasePath()]);
+        if (\strlen($this->baseUrl) > 0) {
+            $url = \vsprintf('%s://%s%s', [$request->getScheme(), $request->getHttpHost(), $request->getPathInfo()]);
+        } else {
+            $url = \vsprintf('%s://%s%s', [$request->getScheme(), $request->getHttpHost(), $request->getBasePath()]);
+        }
```
Comment (Davidmattei) [inline Helper/Environment/Environment.php:89]: Is this not the same?

**Review by Davidmattei — APPROVED** [review]

File: Helper/Environment/Environment.php (hunk 1 above)
Comment (theus77) [reply Helper/Environment/Environment.php]: Fixed

File: Helper/Environment/Environment.php (hunk 2 above)
Comment (theus77) [reply Helper/Environment/Environment.php]: Fixed see commit

**Review by Davidmattei — DISMISSED** [review]

**Review by Davidmattei — APPROVED** [review]

**Pull request merged**